power_grid_model_c/power_grid_model/include/power_grid_model/power_grid_model.hpp

```diff
@@ -85,7 +86,13 @@ constexpr double default_source_rx_ratio = 0.1;
 constexpr double default_source_z01_ratio = 1.0;
 
 // calculation info
-using CalculationInfo = std::map<std::string, double>;
+#if defined(__cpp_lib_hardware_interference_size)
+constexpr size_t cache_line_size = std::hardware_destructive_interference_size;
```

Comment:
I think this one requires glibc 2.28 to work. We must stay on glibc 2.24 to support amazon linux 2.

So maybe remove this one and just set it to 64 as below.

Reply:
i mean, the version test `#if defined(__cpp_lib_hardware_interference_size)` already should already take care of that dependency but i will remove it to keep everything the same between different libraries for now

Reply:
That's not the case. We compile the Python package in 2.28 environment. It just happens that all our codes do not touch 2.28 specific functionality so we can audit the wheel in 2.24 version. 

Adding this `std::hardware_destructive_interference_size` changes that fact.

Reply:
OK, will remove